assets/dev/js/admin/floating-elements/layout.js

```diff
@@ -0,0 +1,27 @@
+import NewFloatingElementsView from 'elementor-admin/floating-elements/view';
+
+export default class extends elementorModules.common.views.modal.Layout {
+	getModalOptions() {
+		return {
+			id: 'elementor-new-floating-elements-modal',
+		};
+	}
+
+	getLogoOptions() {
+		return {
+			title: __( 'New Floating Elements', 'elementor' ),
+		};
+	}
+
+	initialize() {
+		elementorModules.common.views.modal.Layout.prototype.initialize.apply( this, arguments );
```

Comment:
Can you use `super` here?

Reply:
@louiswol94 I refactored this to use super